download-maven-plugin/src/main/java/com/googlecode/download/maven/plugin/internal/cache/Index.java

```diff
@@ -0,0 +1,32 @@
+package com.googlecode.download.maven.plugin.internal.cache;
+
+/**
+ * Cache index.
+ * @author Paul Polishchuk
+ * @since 1.3.1
+ */
+interface Index {
+
+    /**
+     * Adds given path to the index using url parameter as a key.
+     * @param url index key
+     * @param path index value
+     */
+    void put(String url, String path);
```

Comment:
@mickaelistria It's not very abstract actually, as it is located in `internal.cache` package
Do you think we should rename arguments to be more generic?
```java
void put(String key, String value)
```
?